### PR TITLE
a little more accurate Math.cbrt

### PR DESCRIPTION
**Pull request by Yaffle**
fix for #330

### CODE REVIEW

File: es6-shim.js [hunk 1]
```diff
@@ -1269,7 +1269,13 @@
       if (value === 0) { return value; }
       var negate = value < 0, result;
       if (negate) { value = -value; }
-      result = Math.pow(value, 1 / 3);
+      if (value === Infinity) {
```
Comment (ljharb) [inline es6-shim.js]: This looks like the code path could be hit with `-Infinity` too, is that one accurate for the `exp` + `log` path?

Comment (Yaffle) [inline es6-shim.js]: -Infinity is converted to Infinity 2 lines above, so the "true" branch will be involved, and the "sign" will be set before return

Comment (ljharb) [inline es6-shim.js]: ah ha, so it will, thanks

**Comment by ljharb** [comment]
This looks awesome, and your new tests already pass in everything I briefly checked.

Can you rebase this on top of the latest master (rather than merge) with (it looks like from your merge commit) `git rebase paulmillr/master`, and I'll get this merged?

**Comment by Yaffle** [comment]
@ljharb , ok, thanks for the command

**Pull request merged**